x-pack/test/functional/fixtures/kbn_archiver/maps/data.json

```diff
@@ -0,0 +1,912 @@
+{
```

Comment:
Mind if we rename this to `kbn_archiver/maps.json`. We don't need a directory like there was with esArchiver since it's just one file.

Reply:
yes, moved the file and updated the references.